docs/guides/gcp-workspace.md

```diff
@@ -64,7 +64,10 @@ resource "google_project_iam_custom_role" "workspace_creator" {
     "resourcemanager.projects.setIamPolicy",
     "serviceusage.services.get",
     "serviceusage.services.list",
-    "serviceusage.services.enable"
+    "serviceusage.services.enable",
+    "compute.networks.get",
+    "compute.projects.get",
+    "compute.subnetworks.get",
```

Comment:
Apparently, our official doc just asked for either Owner (roles/owner), or Editor (roles/editor) and Project IAM Admin (roles/resourcemanager.projectIamAdmin)...